packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -149,7 +149,8 @@ const getSPARenderer = lazyCachedFunction(async () => {
       _errors: {},
       serverRendered: false,
       data: {},
-      state: {}
+      state: {},
+      once: {}
```

Comment:
I wonder if we use instead a set of ids rather than kv pairs?

Reply:
Yes we should use Set(), was not sure about the hydration with the new json payload